integration-tests/environments/react-native-test-app/package.json

```diff
@@ -0,0 +1,150 @@
+{
+  "name": "@realm/react-native-test-app-tests",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "build:android": "mkdirp dist/res && react-native bundle --entry-file index.js --platform android --dev true --bundle-output dist/main.android.jsbundle --assets-dest dist/res",
```

Comment:
What does the `res` stand for here? (In `dist/res`)

Reply:
We actually just need `mkdir dist` here. The `res` folder will be created if it's missing by the CLI before it copies over Android assets.

Reply:
resource

Reply:
Good to know, thanks @tido64!